docs/operations/blocks-storage.md

```diff
@@ -455,6 +455,20 @@ compactor:
 
 ## Known issues
 
+### Can't ingest samples older than 1h compared to the latest received sample of a tenant
+
+The blocks storage opens a TSDB for each tenant in each ingester receiving samples for that tenant. The received series are kept in the TSDB head and then a new block is cut from the head once the time range period covered by the series in the head reaches the block range period (defaults to 2h).
+
+When a new block is cut from head, the new block contains samples up until -50% of the configured block range period. This means that if you use the default 2h block range period, the head will contain up until 3h series (2h + 50%) and when a new block is cut from head it will contain series up until 1h ago (2h - 50%); technically it's 1h less than the max timestamp in the head.
```

Comment:
Different description of same idea...

New block is cut from the head, when head (in-memory data) covers more than 1.5x of the block range. For 2 hours block range, it means that head needs to have 3h of data to cut a block. Block "start time" is always the minimum sample time in the head, while block "end time" is aligned on block range boundary. New block always covers single "block range" period. Data stored into a block is then removed from the Head. That means that after cutting the block, head will already have at least 1h of data, but possibly even more.

Reply:
> and when a new block is cut from head it will contain series up until 1h ago (2h - 50%)

When I read this few several times, I understood "it" as "block", but in fact it refers to the head.

Reply:
I picked your suggestion and I've done few changes. May you take a look now, please?